rpa-builder/6.1/modules/ROOT/pages/adding-a-pattern-from-screen-capture.adoc

```diff
@@ -0,0 +1,159 @@
+
+image:color-spot-picker-window.png[60%, 60%, The Color Spot Picker window.]
+
+As you move the mouse over the image, the *Live values* section updates showing the following information: 
+
+* *Color*
++
+The color value, expressed in RGB color values (`R:255`, `G:255`, `B:255`), for the current image coordinates. 
+* *Coordinates*
++
+The image coordinates based on the current position of the mouse, indicated in `x;y` coordinates being `0;0` the upper-left corner of the image.
+
+To define the current selection as a Color Spot, either click the left mouse button or press `Insert` or `Space` on the keyboard. 
+
+To delete the Color Spot, use the `Delete` or `Backspace` key.
+
+When you select a Color Spot, it shows in the *Selected Color Spot* section on the left side of the window. 
+
+image:color-spot-picker-window-selected-color.png[60%, 60%, The Color Spot Picker window showing a selected color.]
+
+The following settings lessen the restrictions on Color Spots:
+
+* *Compare Radius (in pixels)* 
++
+Determines the radius around the specified position for the Color Spot in which the specified color must be found. There is a graphical representation of this radius next to the input field.
+* *Color Fuzziness (in bits)* 
++
+Determines the color deviation which is permitted for each RGB color channel. Use this option to specify how many low-order digits in the binary 8-digit color code can be ignored when comparing the color.
+
+[[pattern-strip]]
+== The Pattern Strip Module
+
+Pattern Strip is a UI module that is used across all the wizards for the Search Pattern, Pattern Select, and Pattern Gallery Action Steps. The Pattern Strip consists of a list of Search Patterns, the associated options, and a menu bar. The menu bar buttons vary according to the Action Step and the quantity and selection status of the search patterns.
+
+.Pattern Strip for the Pattern Gallery Action Step:
+image:pattern-strip-pattern-gallery.png[75%, 75%, The Pattern Strip module for the Pattern Gallery Action Step.]
+
+.Pattern Strip for the Pattern Select Action Step:
+image:pattern-strip-pattern-select.png[75%, 75%, The Pattern Strip module for the Pattern Select Action Step.]
+
+Use the mouse to click and select search patterns. A white check mark appears on a blue background in the top-right corner to highlight a selected item. Click again to undo the selection.
+
+Buttons for functions relating only to the selected patterns appear on the menu bar only if at least one pattern is selected.
+
+=== Menu Bar
+
+The Pattern Strip menu bar has general buttons that show in all related Action Steps and buttons that are specific to the Search Pattern, Pattern Select, and Pattern Gallery Action Steps. The following table shows the button images, names, descriptions, and also in which of the related Action Steps they are available:  
```

Comment:
The following table shows the button images, names, and descriptions, and the Action Steps in which they are available: